src/components/board.js

```diff
@@ -2,34 +2,68 @@ import React from 'react';
 import Card from './Card/card';
 class Board extends React.Component {
     constructor(props) {
-        super(props);
+        super(props)
+        this.change = this.change.bind(this);
```

Comment:
if I'm not wrong, we can use arrow operator (`()=>{}`) while creating functions, and no need to bind this. I can be wrong too. :)